lib/tapjoy/autoscaling_bootstrap/configure_autoscaler.rb

```diff
@@ -16,6 +20,11 @@ def initialize(misc_config:, aws_env:, user_data:, **unused_values)
 
       private
 
+      # Check if ASG exists so we know whether to create or update
+      def autoscaling_group_exists
+        !Tapjoy::AutoscalingBootstrap::AWS::Autoscaling::Group.describe.nil?
```

Comment:
This reads like a double-negative and so is a little hard to parse. Can it be restated with `unless` or something?